### PR TITLE
pop quiz added in (docs/comb-msi/mux-demux.md)

### PR DESCRIPTION
**Pull request by ayan-biswas0412**
A meaningful title for PR, not like: made changes to xyz.md

Fixes #505 

#### Changes done:
- [x]  pop quiz added in (docs/comb-msi/mux-demux.md)

#### Screenshots
image

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by ayan-biswas0412** [comment]
@Shivansh2407 can you please Review this

**Review by Shivansh2407 — CHANGES_REQUESTED** [review]
Awesome Work Done @ayan-biswas0412 . I would here again want you to keep the options in a single line rather than entering a new line.

**Comment by ayan-biswas0412** [comment]
> Awesome Work Done @ayan-biswas0412 . I would here again want you to keep the options in a single line rather than entering a new line.

yes I have the same question in mind too , but will there be any option to doing that like there are two possibilities I have to change the codebase for the styling or I have to give smaller options such that they fit but again in mobile devices they will be in multiline , so what can I do now ?

**Comment by ayan-biswas0412** [comment]
@Shivansh2407 I have fixed the multiline options for this PR 
Screenshot
image

File: docs/comb-msi/mux-demux.md [hunk 1]
```diff
@@ -158,3 +158,32 @@ A de-multiplexer is equivalent to a single pole multiple way switch as shown in
...
 A 1 : 16 demultiplexer can be implemented using **two** 1 : 8 demultiplexers.
 <iframe width="100%" height="400px" src="https://circuitverse.org/simulator/embed/44796" id="demux_04" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen> </iframe>
+
+{:.quiz}
+
+1. Which of the following directs data from input to a selected output line ?
+   1. Demultiplexer
+   * Multiplexer
+   * Coder
+   *  Both MUX and DEMUX
```
Comment (manjotsidhu) [inline docs/comb-msi/mux-demux.md:168]: For conserving more space
```suggestion
   *  Both MUX & DEMUX
```

**Review by manjotsidhu — APPROVED** [review]

**Review by ayan-biswas0412 — COMMENTED** [review]
all necessary changes done

**Review by manjotsidhu — APPROVED** [review]

**Review by Shivansh2407 — APPROVED** [review]
Great work @ayan-biswas0412 🎉

**Pull request merged**